### ISSUE TITLE
[RAT-Profile] (Darkmode) QR-Code should have a black background

### ISSUE THREAD

**Issue by jucktnich** [corona-warn-app/cwa-wishlist#511]
## Current Implementation

The QR-Code has has a white background on light- and darkmode.

## Suggested Enhancement

Use a dark background. 

## Expected Benefits

Just looks better IMO.

![image](https://user-images.githubusercontent.com/73582096/117982351-35f84100-b336-11eb-9667-611ccd5edb4b.png)

## Other

(An inversion of the qr-code picture should be easy to implement)

**Comment by Ein-Tim** [comment]
@jucktnich Are you only asking for the RAT Profile QR-Code or also for the event-registration QR-Code? 
If you are only asking for the RAT Profile, why don't you think an implementation for the event-registration should be considered?

**Comment by jucktnich** [comment]
See #512

**Comment by Ein-Tim** [comment]
@jucktnich I saw #512 but it does not mention dark mode. The new title is better. Thank you!

**Comment by jucktnich** [comment]
Yeah, a bit confusing

**Comment by Ein-Tim** [comment]
Now it's clear, thanks! Supporting this btw!

**Comment by heinezen** [comment]
@jucktnich 

Thank you for the request. We have created  [EXPOSUREAPP-7301](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-7301) in our internal Jira for ypur proposal. The devs will discuss the possibility now.

----
Corona-Warn-App Open Source Team

**Comment by AdamVB** [comment]
Yesterday I was in the Airport in Malta and the Health Authority there had Issues scanning the Vaccination Certificate while the Phone (iOS) was in dark mode (max brightness). After setting it to light mode their app could scan the Vaccination Cert right away.
So maybe a black background is not always the best idea. This might need more testing.

**Comment by dsarkar** [comment]
@AdamVB Thanks for the info. We will forward this internally.


---
Corona-Warn-App Open Source Team

**Comment by jucktnich** [comment]
@AdamVB the qrcode wasn't inverted right?

**Comment by AdamVB** [comment]
no, we tried it in all angles, even putting paper above both phones to avoid light reflexions but none of that helped until a colleague came by and gave the hint that turning on the "light" mode works a lot better. And as soon as we did that it scanned instantly.

**Comment by jucktnich** [comment]
@AdamVB you used the CWA right?

**Comment by AdamVB** [comment]
yes, displayed the QR Code from within CWA,

but i don't know what App the maltese authorities were using. But they all had the same phones/devices for scanning.

**Comment by jucktnich** [comment]
Yeah, but then this has nothing to do with my issue, I think, since my proposal wasn't implemented yet. Or do I miss something?

**Comment by AdamVB** [comment]
Well right now the background is dark grey so maybe it is already implemented?

**Comment by jucktnich** [comment]
My proposal is to invert the picture of the qr-code, not make the background black, since as you can see, this is and was already implemented.

**Comment by AdamVB** [comment]
ohh, i thought you meant the borders and background around the QR code, not the QR code content itself. Ok then my concerns are not in the right place here.
